src/program/symlink.py

```diff
@@ -217,7 +208,6 @@ def _symlink(self, item: Union[Movie, Episode]) -> bool:
         extension = os.path.splitext(item.file)[1][1:]
         symlink_filename = f"{filename}.{extension}"
         destination = self._create_item_folders(item, symlink_filename)
-        source = os.path.join(self.rclone_path, item.folder, item.file)

```

Comment:
Removed because the logic is different to `_get_item_path`, which I use instead now.